### ISSUE TITLE
Guide on how to roll out sqlfluff on a new project/team

### ISSUE THREAD

**Issue by alanmcruickshank** [sqlfluff/sqlfluff#286]
Referenced in #265 

First:
> It's a new structure and I don't want to trigger any negative stereotypes about engineers caring too much about unimportant details and slowing down the ML work, if that makes sense. If it were just me or just engineers, I'd feel less nervous.

Then
> Full transparency: We're not using sqlfluff at all in production at tails.com yet. I'm using it to clean up a few projects, and I'm almost ready to put it in the CI pipeline for one part of one project. I think there's probably a place for a guide about how to roll out sqlfluff on a project without pissing people off. 

Starting to use sqlfluff on a new project is probably daunting. There should be a short how-to on how to do it without pissing people off (...too much). This is a placeholder for that.

Anybody who has thoughts on this - feel free to drop comments into here (including you @barrywhart  😄 ).

**Comment by barrywhart** [comment]
About rolling out SQLFluff to the team:
First I just talked about it and added it to one or two projects I was working on myself.
I integrated with the build, so people can see the report, but it doesn't make the build fail.
I made the first few PRs myself (to not burden others with the work) but asked them to review the PRs. Initially, I mostly focused on whitespace changes, which can be easily hidden in GitHub PR review. This makes the PRs easy to review and helps assure they're not going to break existing code. It's also a good way to get people looking at and thinking about formatting and readability.
I've added other projects on an "opt-in" basis. One person has been a big adopter. He's working on a project with a new data scientist whose formatting was very different, so he appreciated the standardization. I also worked to assure him the changes are pretty "common sense", not too onerous.
